mantle/platform/qemu.go

```diff
@@ -1398,8 +1398,8 @@ func baseQemuArgs(arch string, memoryMiB int) ([]string, error) {
 		ret = []string{
 			"qemu-system-ppc64",
 			// kvm-type=HV ensures we use bare metal KVM and not "user mode"
-			// https://qemu.readthedocs.io/en/latest/system/ppc/pseries.html#switching-between-the-kvm-pr-and-kvm-hv-kernel-module
-			"-machine", "pseries,kvm-type=HV," + machineArg,
+			// https://www.qemu.org/docs/master/system/ppc/pseries.html
+			"-machine", "pseries,kvm-type=HV,ic-mode=xics," + machineArg,
```

Comment:
Can you clarify why not use `XIVE`? Or it is suppose to work only with LPARs?
Seems XICS is for P8 and  and XIVE  for P9 and P10?

Reply:
For KVM-HV hypervisor running in a pseries-guest native xive mode is currently not supported, but rather native xics.
Qemu by default will attempt enabling native xive for pseries guests first and if that fails , fallsback to xics based interrupts hence this warning is emitted.

Explicitly asking Qemu to use xics base interrupt controller forces qemu to not attempt to use XIVE interrupt controller preventing this warning message from being emmited.